tsconfig.web.json

```diff
@@ -5,7 +5,7 @@
         "module": "es2020",
         "target": "es2020",
         "jsx": "react",
-        "lib": [ "webworker", "ES2020", "DOM" ],
+        "lib": [ "ES2020", "DOM" ],
```

Comment:
Why did you need to remove this? No adverse side effects?

Reply:
It turns out when we add "webworker" in lib TS is treating everything as workers. The result is that here I use types for the navigator.wakeLock which are adding the wakeLock prop to the Navigator interface but we get an error when "webworker" lib is added in the config because the type of navigator is WorkerNavigator. It seems to me that if we need the webworker lib option we will need to add some extra care about making TS to use the option only for the workers somehow.

It doesn't seems to have any adverse effects with the existing code base. Maybe now the workers won't be so restricted by TS but I haven't tested it. Anyway treating all files as workers doesn't sound correct too.